src/firmware/app/primitives/autochip_move_primitive.c

```diff
@@ -0,0 +1,48 @@
+#include "firmware/app/primitives/autochip_move_primitive.h"
+
+#include <math.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "firmware/app/primitives/move_manager.h"
+
+void app_autochip_move_primitive_start(AutochipMovePrimitiveMsg prim_msg,
+                                       void* void_state_ptr, FirmwareWorld_t* world)
+{
+    app_move_manager_start(void_state_ptr, world, prim_msg.position_params,
+                           prim_msg.final_angle.radians);
+
+    const FirmwareRobot_t* robot = app_firmware_world_getRobot(world);
+
+    Dribbler_t* dribbler = app_firmware_robot_getDribbler(robot);
+    Chicker_t* chicker   = app_firmware_robot_getChicker(robot);
+    app_dribbler_setSpeed(dribbler, (uint32_t)prim_msg.dribbler_speed_rpm);
+    app_chicker_enableAutochip(chicker, prim_msg.chip_distance_meters);
+}
+
+static void autochip_move_end(void* void_state_ptr, FirmwareWorld_t* world)
+{
+    app_move_manager_end(void_state_ptr, world);
+    FirmwareRobot_t* robot = app_firmware_world_getRobot(world);
+
+    Chicker_t* chicker = app_firmware_robot_getChicker(robot);
+    app_chicker_disableAutochip(chicker);
+    app_chicker_disableAutokick(chicker);
+
+    Dribbler_t* dribbler = app_firmware_robot_getDribbler(robot);
+    app_dribbler_setSpeed(dribbler, 0);
+}
+
+static void autochip_move_tick(void* void_state_ptr, FirmwareWorld_t* world)
```

Comment:
Missing `app_` prefix here

Reply:
Similar for a few functions below. Could you do a pass through all the functions in this PR?